pkg/resources/schema.go

```diff
@@ -213,73 +213,90 @@ func CreateContextSchema(ctx context.Context, d *schema.ResourceData, meta any)
 	database := d.Get("database").(string)
 	id := sdk.NewDatabaseObjectIdentifier(database, name)
 
-	dataRetentionTimeInDays,
-		maxDataExtensionTimeInDays,
-		externalVolume,
-		catalog,
-		replaceInvalidCharacters,
-		defaultDDLCollation,
-		storageSerializationPolicy,
-		logLevel,
-		traceLevel,
-		suspendTaskAfterNumFailures,
-		taskAutoRetryAttempts,
-		userTaskManagedInitialWarehouseSize,
-		userTaskTimeoutMs,
-		userTaskMinimumTriggerIntervalInSeconds,
-		quotedIdentifiersIgnoreCase,
-		enableConsoleOutput,
-		err := GetAllDatabaseParameters(d)
-	if err != nil {
-		return diag.FromErr(err)
-	}
+	create := true
```

Comment:
TBH I would prefer an explicit separation here, i.e.:
- if name is not PUBLIC -> run create
- otherwise if does not exist -> run create
- otherwise -> run update

Reply:
Simplified a bit.

Reply:
It's not a separation like I asked for. Now the execution path is still not clear.

Let's leave it like this, and will follow up and do it in the following PRs. I will keep it open for the time-being.